src/dropin.js

```diff
@@ -439,6 +439,28 @@ Dropin.prototype.updateConfiguration = function (property, key, value) {
   this._navigateToInitialView();
 };
 
+/**
+ * Get a list of the available payment methods presented to the user. This is useful for knowing if a paricular payment option was presented to a customer that is browser dependant such as Apple Pay, Google Pay and Venmo. Returns an array of strings. Possible values:
```

Comment:
Is there a limitation preventing them from calling `getAvailablePaymentOptions()` before the drop-in UI appears?

Reply:
Shouldn't be a problem, since you can't access this method until the instance is ready.